x/quarantine/spec/04_events.md

```diff
@@ -3,30 +3,31 @@
 The `x/quarantine` module emits the following events.
 
 <!-- TOC -->
-  - [EventOptIn](#eventoptin)
-  - [EventOptOut](#eventoptout)
-  - [EventFundsQuarantined](#eventfundsquarantined)
-  - [EventFundsReleased](#eventfundsreleased)
+
+- [EventOptIn](#eventoptin)
+- [EventOptOut](#eventoptout)
+- [EventFundsQuarantined](#eventfundsquarantined)
+- [EventFundsReleased](#eventfundsreleased)
 
 ## EventOptIn
 
 This event is emitted when an account opts into quarantine.
 
 `@Type`: `/cosmos.quarantine.v1beta1.EventOptIn`
 
-| Attribute Key | Attribute Value                      |
-|---------------|--------------------------------------|
-| to_address    | {bech32 string of account opting in} |
+| Attribute Key | Attribute Value                        |
+| ------------- | -------------------------------------- |
+| to_address    | \{bech32 string of account opting in\} |
```

Comment:
This is incorrect syntax for markdown tables. the dashes are supposed to go all the way between the pipes.

Reply:
The formatter I have in VS Code automatically did that. I've copied and pasted it here... seems like it works like that.

| Attribute Key | Attribute Value                        |
| ------------- | -------------------------------------- |
| to_address    | \{bech32 string of account opting in\} |

Reply:
A lot of markdown interpreters understand it, but the actual syntax definition says to have them go all the way across, so it's not guaranteed to work.

Reply:
Updated